src/lightning-stubs/uiGraphQLApi/uiGraphQLApi.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { createTestWireAdapter } from '@salesforce/wire-service-jest-util';
+
+export class graphql extends createTestWireAdapter() {
+    static emit(value, filterFn) {
+        super.emit({ data: value, errors: undefined }, filterFn);
```

Comment:
Is this required? `errors: undefined`. The default value for unset object keys is `undefined`

Reply:
I'm not really sure - but it's what our stub in lightning-components has